sdk/storage/Azure.Storage.Files.DataLake/api/Azure.Storage.Files.DataLake.netstandard2.0.cs

```diff
@@ -228,6 +231,7 @@ public partial class DataLakePathClient
         public DataLakePathClient(string connectionString, string fileSystemName, string path) { }
         public DataLakePathClient(string connectionString, string fileSystemName, string path, Azure.Storage.Files.DataLake.DataLakeClientOptions options) { }
         public DataLakePathClient(System.Uri pathUri) { }
+        public DataLakePathClient(System.Uri pathUri, Azure.AzureSasCredential credential, Azure.Storage.Files.DataLake.DataLakeClientOptions options = null) { }
```

Comment:
We have a different ctor parameter pattern for the TokenCredential (3 overloads instead of defaulted options). Is this intentional?

Reply:
Less code. I was trying to understand why do we have such pattern in place, but nobody would recall why. From user POV there shouldn't be a difference (unless I'm missing something subtle here).

You can observe similar inconsistencies in APIs implementations. Some would come with multiple overloads, some will use optional parameters. I think those with multiple overloads approach are older.

Reply:
I think the goal of all of these is to make mainline APIs look simpler. We've noticed that customers get scared of methods with many parameters even if they are all optional so opted for having overloads in important scenarios.

Reply:
Gotcha. I'll make it consistent then.